app/schemas/report.py

```diff
@@ -11,3 +12,13 @@ class ReportResponse(BaseModel):
     scrape_date: datetime
     report_count: int
     score: float
+
+
+class ProcessedReportResponse(BaseModel):
+    id: str
+    user: dict[str, Any]
+    label: int
+
+
+class ReportProcess(BaseModel):
+    method: Literal["approve", "reject"]
```

Comment:
Do we still  need this?

Reply:
for `POST` request body?

Reply:
We don't implement reject anymore, right?

Reply:
yea, but the frontend Reject button still need a body schema to send request right?

Reply:
I don't undersand why you don't just create 2 separate endpoints at the first place

Reply:
u mean put these "ProcessedReport" in a different file, right? :V

I wanted to use `tweepy.modals.User` at first for short implementation 😅 instead of specify a long schema, so i put in the same file. But it won't let me do that :))

Reply:
but can u think of a better name than proccessed report :V

Reply:
I don't understand what you are saying. I mean create 2 API endpoints for approving and rejecting report, instead of putting `approve` and `reject` in request body.

Reply:
oh okay, misunderstand :V.